scripts/publish.js

```diff
@@ -56,13 +56,25 @@ if (checkTags.includes(DOCKER_IMAGE_VERSION)) {
 }
 
 // Proceed to build using the right CLI version
-const dockerBuildExitCode = shell.exec(
-  `docker build --build-arg SALESFORCE_CLI_VERSION=${SALESFORCE_CLI_VERSION} --tag ${DOCKER_HUB_REPOSITORY}:${DOCKER_IMAGE_VERSION} .`
+
+/* SLIM VERSION */
+const slim_dockerBuildExitCode = shell.exec(
+  `docker build --file ./dockerfiles/Dockerfile_slim --build-arg SALESFORCE_CLI_VERSION=${SALESFORCE_CLI_VERSION} --tag ${DOCKER_HUB_REPOSITORY}:${DOCKER_IMAGE_VERSION}-slim .`
+);
+/* FULL VERSION */
+const full_dockerBuildExitCode = shell.exec(
+  `docker build --file ./dockerfiles/Dockerfile_full --build-arg SALESFORCE_CLI_VERSION=${SALESFORCE_CLI_VERSION} --tag ${DOCKER_HUB_REPOSITORY}:${DOCKER_IMAGE_VERSION}-full .`
 );
 
 // Push to the Docker Hub Registry
-const dockerPushExitCode = shell.exec(
-  `docker push ${DOCKER_HUB_REPOSITORY}:${DOCKER_IMAGE_VERSION}`
+
+/* SLIM VERSION */
+const slim_dockerPushExitCode = shell.exec(
+  `docker push ${DOCKER_HUB_REPOSITORY}:${DOCKER_IMAGE_VERSION}-slim`
+);
+/* FULL VERSION */
+const full_dockerPushExitCode = shell.exec(
+  `docker push ${DOCKER_HUB_REPOSITORY}:${DOCKER_IMAGE_VERSION}-full`
```

Comment:
If we end up having more than two images we may want to consider a more dynamic way to build and publish the images but with only two this seemed like the simplest solution while still running only one publish script. `scripts/publish.js`.

Reply:
Formatting for the tags follows Node here https://hub.docker.com/_/node